exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -54,6 +59,13 @@
   static final Set<MetricDataType> DOUBLE_TYPES = ImmutableSet.of(DOUBLE_GAUGE, DOUBLE_SUM);
   private static final int MIN_TIMESTAMP_INTERVAL_NANOS = 1000000;
 
+  // Mapping outlined at https://cloud.google.com/monitoring/api/resources#tag_gce_instance
+  private static final Map<String, AttributeKey<String>> gceMap =
```

Comment:
This might be more simple (and java-8 friendly) expressed as:
```
private static AttributeKey<String> gceToSemantic(String gce) {
  switch(gce) {
    case "project_id": return SemanticAttributes.CLOUD_ACCOUNT_ID;
    ...
    default: return null;
  }
```
That's byte-code compatible with java 8 (just not source).